### DISCUSSION TITLE
Initialising Screen

### DISCUSSION THREAD

**Discussion by Smart-Techno1ogy** [Blackymas/NSPanel_HA_Blueprint#219] [General]
Happy weekend all, I have successfully flashed my panels and have then all live in ESPHome showing as online and auto discovered by HA

I have updated the TFT's as described but am now stuck with the panels simply displaying a 3d HA Graphic stating "initialising". How  do you get passed this? what steps have I missed? 

Thanks

**Comment by ghost** [comment]
This is normally due to the name in ESP home containing a special character or '-', or HA Blueprint hasn't been installed / the name of the device is incorrect in the blueprint.

**Comment by bahuta88** [comment]
I´m currently having the same issue.
For me it seems to be related to the API PWD.
I was not able to change it, so I readded the nspanel as new device in ESPHome.
That leads to the fact, that the encryptio is being enabled as a default and no option to disable and no option to enter the encryption salt to the nspanel_esphome.yaml as it is not locally on the HA.
For now I also don´t now what to do.
![image](https://user-images.githubusercontent.com/123777222/215204564-7c17872a-25ef-4f16-8058-2b8bc154e7ce.png)

**Comment by edwardtfn** [comment]
I'm closing this discussion as we have released v4.0 (#1057) and this shouldn't be an issue anymore.
Please add an issue or a feature request for the missing things.
